src/lwc/rd2ElevateCreditCardForm/rd2ElevateCreditCardForm.html

```diff
@@ -27,17 +27,16 @@ <h1 class='slds-text-body_regular' data-qa-locator='heading'>{labels.elevateWidg
                         </div>
                     </lightning-layout-item>
 
-                    <template if:false={isDisabled}>
-                        <lightning-layout-item padding="horizontal-small" size="6">
-                            <lightning-input name="cardholderName"
-                                             label={labels.cardholderNameLabel}
-                                             data-qa-locator="input Elevate Cardholder Name"
-                                             data-id="cardholderName"
-                            ></lightning-input>
-                        </lightning-layout-item>
-                    </template>
-
                     <lightning-layout-item size='12' padding='around-small'>
+                        <template if:false={isDisabled}>
```

Comment:
@npsp-reedestockton ,  after the changes, the Cardholder Name field takes the whole width of the form. 

Also, it seems that bottom margin/padding of the Elevate iframe has increased, so we might either need to add padding above Cardholder Name or decrease the bottom padding of the `lightning-layout-item`. 

Also, when an error is issued, there is no spacing between the Cardholder Name. The following code replaces the `lightning-layout-item` on line 30 incorporating the feedback above:

```

                    <lightning-layout-item size="12" multiple-rows="true" class='slds-p-horizontal_small slds-p-top_small'>

                        <template if:true={isDisabled}>
                            <p class='fade-in'>
                                <span class='slds-content-message'>
                                    <lightning-formatted-rich-text value={labels.elevateDisabledMessage}
                                        data-qa-locator="richtext Elevate Disabled Message">
                                    </lightning-formatted-rich-text>
                                </span>
                                <lightning-button variant='base' label={labels.elevateEnableButtonLabel}
                                    title={labels.elevateEnableButtonLabel} onclick={handleUserEnabledWidget}
                                    class='slds-m-left_xx-small' data-qa-locator='button Use Elevate Now'>
                                </lightning-button>
                            </p>
                        </template>

                        <template if:false={isDisabled}>
                            <div if:true={alert.show} class='slds-p-bottom_medium'>
                                <c-util-alert-banner if:true={alert.show} theme={alert.theme} variant={alert.variant}
                                    icon={alert.icon} message={alert.message} data-qa-locator='alert Message'>
                                </c-util-alert-banner>
                            </div>

                            <div class="slds-col slds-size_1-of-2">
                                <lightning-input name="cardholderName" label={labels.cardholderNameLabel}
                                    data-qa-locator="input Elevate Cardholder Name" data-id="cardholderName">
                                </lightning-input>
                            </div>

                            <div class='iframe-container' data-qa-locator='iframe Elevate'>
                                <template if:true={isLoading}>
                                    <lightning-spinner alternative-text={labels.spinnerAltText}>
                                    </lightning-spinner>
                                </template>
                                <iframe class='payment-services-iframe' title={labels.elevateWidgetLabel}
                                    src={tokenizeCardPageUrl} data-id={labels.elevateWidgetLabel}>
                                </iframe>
                            </div>
                        </template>
                    </lightning-layout-item>

```

The screenshot after code is changed per above feedback:
<img width="825" alt="Screen Shot 2020-12-02 at 11 23 06 PM" src="https://user-images.githubusercontent.com/28743286/100977115-5e615100-34f5-11eb-99af-fe959cea746b.png">

<img width="831" alt="Screen Shot 2020-12-02 at 11 21 20 PM" src="https://user-images.githubusercontent.com/28743286/100976970-26f2a480-34f5-11eb-85b9-6e8d0e2d9227.png">

Reply:
Thanks for that.  Now, though, the spacing below the message when not using Elevate is insufficient.

Reply:
ok, feel free to see how it can work for all scenarios with regards to spacing below, but I think the rest should be fine.